datasimulator/file_handling.py

```diff
@@ -0,0 +1,20 @@
+from .utils import attempt  # same as datasimulator.utils
+from cdislogging import get_logger
+logger = get_logger("data-simulator", log_level="info")
+
+
+def write_node_to_file(outfile, node):
+    outfile.write(f"{node.name}\t{node.category}\n")
+
+
+def write_to_file_or_log_error(file_path, submission_order):
```

Comment:
```suggestion
def write_submission_order_to_file_or_log_error(file_path, submission_order):
```
this is a bit nitpicky, but imo if the function is not generic, the function name shouldn't be either

Reply:
yeah if it didn't have the submission param there may be something to generalize, but it would be better to have a specific name like this